src/object/srv_obj_migrate.c

```diff
@@ -2227,7 +2238,7 @@ migrate_one_epoch_object(daos_epoch_range_t *epr, struct migrate_pool_tls *tls,
 		unpack_arg.invalid_inline_sgl = 0;
 		rc = dsc_obj_list_obj(oh, epr, NULL, NULL, NULL,
 				     &num, kds, &sgl, &anchor,
-				     &dkey_anchor, &akey_anchor, &csum);
+				     &dkey_anchor, &akey_anchor, p_csum);
```

Comment:
sorry, I lost the point why EC need to re-calculate the csum and replica obj need not? could you please remind me, thanks

Reply:
for replicate object rebuild will just copy the  data + csum from the remote replicate for inline data, but for EC object, rebuild need recalculate the data, so it does not need get the csum during enumeration anymore, which also cause some issues.